src/antlr4/Cypher.g4

```diff
@@ -17,7 +17,10 @@ oC_Cypher
     : SP ? oC_AnyCypherOption? SP? ( oC_Statement | kU_DDL | kU_CopyCSV ) ( SP? ';' )? SP? EOF ;
 
 kU_CopyCSV
-    : COPY SP oC_SchemaName SP FROM SP kU_FilePaths ( SP? '(' SP? kU_ParsingOptions SP? ')' )? ;
+    : COPY SP oC_SchemaName kU_CopyCSV_Properties? SP FROM SP kU_FilePaths (SP? ',' SP? kU_FilePaths)* ( SP? '(' SP? kU_ParsingOptions SP? ')' )? ;
```

Comment:
Let's do a different rule for COPY NPY since it has a different interpretation of multiple files (aligned on column), so let's have

kU_CopyCSV (btw open an issue about renaming all CopyCSV to Copy because we are copying different file formats now) as it is and

kU_CopyNPY whose grammar is
```
COPY SP oC_SchemaName SP FROM SP kU_FilePaths
```

We don't specify columns any more, instead we expect file paths to match all columns as defined in DDL (so that we have the same constraint as copy)